webcompat/form.py

```diff
@@ -380,6 +382,18 @@ def domain_name(url):
     return domain
 
 
+def switch_problem_subtype(type):
```

Comment:
[`type`](https://docs.python.org/3/library/functions.html#type) is a reserved keyword in Python. Choose another word.